<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-UC5TO5S" failures="1" tests="1" name="Login and create account error message" time="12.883" errors="0" timestamp="2020-11-23T20:31:16 IST">
  <testcase classname="assessmentpackage.TC_02Test" name="validate_error_msg" time="12.808">
    <failure type="java.lang.AssertionError" message="Error message is displayed when email ID is invalid during sign in expected [false] but found [true]">
      <![CDATA[java.lang.AssertionError: Error message is displayed when email ID is invalid during sign in expected [false] but found [true]
at assessmentpackage.login_signup_bug.validate_error_message(login_signup_bug.java:82)
at assessmentpackage.TC_02Test.validate_error_msg(TC_02Test.java:20)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 30 stack frames]]>
    </failure>
  </testcase> <!-- validate_error_msg -->
</testsuite> <!-- Login and create account error message -->
